net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml

```diff
@@ -3,6 +3,7 @@
     <General VisibleName="General" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/general/index" order="1"/>
     <RIP VisibleName="RIP" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/rip/index" order="10" />
     <OSPF VisibleName="OSPF" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf/index" order="20" />
+    <OSPF6 VisibleName="OSPFv3" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf6/index" order="25" />
```

Comment:
@fabianfrz First time I saw these :) I love the globe for BGP 👍 
RIP fits very good, it's like it's own routing design.
OSPF could also be http://fontawesome.io/icon/connectdevelop/ as there can be many paths to the same direction